solution.js

```diff
@@ -0,0 +1,63 @@
+class Humans {
+    constructor (name, age){
+        this.name = name
+        this.age = age
+    }
+    
+ageOneyear(){
+    this.age = this.age + 1
+    return this.age
+}
+
+eating (){
+    return console.log("mmm, mmm, mmm, I'm love'n it")
+}
+}
+
+class Gender extends Humans{
+    constructor (name,age,gender)
+    {
+    super (name, age)
+    this.gender = gender
+    }
+}
+
+let kenny = new Humans ("Kenny", 28)
+let dough = new Humans("Dough", 18)
+
+console.log(kenny.age)
+
+console.log(kenny.ageOneyear())
+
+console.log(kenny.eating())
+
+
+//Q2
+class Vector {
+    constructor (x , y){
+        this.x = x
+        this.y = y
+    }
+
+    plus(anotherVector){
+        return new Vector (this.x + anotherVector.x , this.y + anotherVector.y)
+    }
+
+    minus(anotherVector) {
+        return new Vector(this.x - anotherVector.x, this.y - anotherVector.y)
+
+    }
+    getLength() {
+        return Math.hypot(this.x,this.y)}
+    
+}
+var v1 = new Vector(1, 2)
+var v2 = new Vector(2, 3)
+
+console.log(v1)
+console.log (v2)
+
+console.log(v1.plus(v2));
+
+var v3 = new Vector(3, 4)
+console.log(v3.getLength())
```

Comment:
Gold star for clear argument names. Argument and variable names should always be clear about what the value actually is.